apis/placement/v1alpha1/stagedupdate_types.go

```diff
@@ -17,6 +17,13 @@ import (
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope=Cluster,categories={fleet,fleet-placement},shortName=crsur
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:printcolumn:JSONPath=`.spec.placementName`,name="Placement",type=string
+// +kubebuilder:printcolumn:JSONPath=`.spec.resourceSnapshotIndex`,name="ResourceSnapshot",type=string
```

Comment:
have a question about this field. 

we want an index or a name? from the example you provided, it's a name.

The resources could be divided to multiple snapshots.

Reply:
It's the name. And we have a validation that only the master snapshot name should be provided.

Reply:
Then we need to rename the spec field as discussed

Reply:
As per discussion, we will keep the APIs using index, and I'll adjust implementation in a separate PR.